packages/react-native/ReactCommon/jsc/JSCRuntime.cpp

```diff
@@ -377,11 +377,13 @@ JSCRuntime::JSCRuntime(JSGlobalContextRef ctx)
 {
 #ifndef NDEBUG
 #ifdef _JSC_HAS_INSPECTABLE
+#if (__OSX_AVAILABLE_STARTING(MAC_NA, IPHONE_16_4))
```

Comment:
Original author of `_JSC_HAS_INSPECTABLE`, should we move this check to the macros' definition? I can do so in my already open PR where I refactor some of it https://github.com/facebook/react-native/pull/39549

Reply:
Hi @Saadnajmi, yes, we can. I just saw we had a check on the iOS version at line 303, and I think that that's what triggered the issue... The version is set to `160400`, which is... 160.4!!!
I think it should be `16400` if we want to specify 16.4.

```diff
- #if __IPHONE_OS_VERSION_MAX_ALLOWED >= 160400
+ #if __IPHONE_OS_VERSION_MAX_ALLOWED >= 16400
```

for example, the [check for iOS 17, is `17000`](https://github.com/facebook/react-native/blob/767197c9a2578333c9bb641ccf3a87af456f3f82/packages/react-native/React/Views/RCTView.m#L899).

Reply:
Damn. I remember double checking that. Ok, I guess we don't need the double check then, and I'll fix the versions.

Reply:
Also closing this thread so we can keep discussion in https://github.com/facebook/react-native/pull/39549#discussion_r1578074116 , but as noted there, I think the existing macro is correct